styled-components.tsx

```diff
@@ -0,0 +1,110 @@
+import { Fragment } from 'react';
+import React from 'react';
+import styled from 'styled-components/native';
+import { ActivityIndicator } from 'react-native';
+
+export const Title = styled.Text`
+    color: #000000;
+    font-size: 24px;
+    font-weight: bold;
+    margin: 20px;
+`;
+
+export const FormLabel = styled.Text<{invalidInput: boolean}>`
+    color: ${(props) => props.invalidInput ? "#C11A26" : "#777777"};
+    font-size: 12px;
+    font-weight: normal;
+    margin-vertical: 12px;
+    margin-horizontal: 20px;
+`;
+
+const FormInput = styled.TextInput<{invalidInput: boolean}>`
+    border: 1px;
+    border-radius: 8px;
+    border-color: ${(props) => props.invalidInput ? "#C11A26" : "#777777"};
+    height: 40px;
+    padding-left: 10px;
+    margin-horizontal: 20px;
+`;
+
+const ButtonContainer = styled.TouchableOpacity`
+    height: 44px;
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    justify-content: space-evenly;
+    margin-top: 12px;
+    margin-horizontal: 20px;
+    border-radius: 8px;
+    background-color: ${(props) => props.disabled ? "#FEB800" : "#841584"};
+`;
+
+const ButtonText = styled.Text`
+    color: #FFFFFF;
+    font-size: 16px;
+    font-weight: normal;
+`;
+
+const OptionContainer = styled.TouchableOpacity<{onPress?: () => void, option: string, selectedOption: string}>`
+    height: 35px;
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    justify-content: space-evenly;
+    margin-top: 10px;
+    margin-horizontal: 20px;
+    border-radius: 8px;
+    background-color: ${(props) => props.option == props.selectedOption ? "#000000" : "#FFFFFF"};
```

Comment:
```suggestion
    background-color: ${(props) => props.option === props.selectedOption ? "#000000" : "#FFFFFF"};
```

A gente evita usar `==` porque ele compara apenas a "veracidade" dos dois valores. O `===` compara tanto valor quanto tipo, o que deixa a comparação mais precisa e com menos chance de side-effects. Mais detalhes [nessa página](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness) ou em outras pesquisas na internet.